lib/bap/bap_self.mli

```diff
@@ -10,4 +10,31 @@ module Create() : sig
   val info    : ('a,formatter,unit) format -> 'a
   val warning : ('a,formatter,unit) format -> 'a
   val error   : ('a,formatter,unit) format -> 'a
+
```

Comment:
I usually try not to duplicate the comments. As they will start to diverge very soon :)